scorer/verifications/seed_connected.py

```diff
@@ -1,54 +1,61 @@
 from arango import ArangoClient
 import time
-import sys
-sys.path.append('..')
-import config
 
+SEED_CONNECTION_LEVELS = ['just met', 'already known', 'recovery']
```

Comment:
except `suspicious` and `reported` so if a seed is connected to someone with any of `['just met', 'already known', 'recovery']` they become `SeedConnected`
yes, If we want to change that we should do another release.
I think releasing scorer is not as risky as other services and if we decide to change something it can be done in a day.